### ISSUE TITLE
Possible conflict with Simple Calender and workflow initialization issue

### ISSUE THREAD

**Issue by maxobremer** [fantasycalendar/FoundryVTT-RestRecovery#113]
When I have this module active I cant rest as a dm, nor player. the rest button in the bottom left underneath the player list doesnt even show up for players (does for gm and it should for players as well, in regards to the settings)

tried to rest with the console open and got this:
![image](https://user-images.githubusercontent.com/86924581/208297826-09b67fa9-7ea5-49e9-bcfb-53270f0e14e8.png)

then i tried resting again without simple calender and got this:
![image](https://user-images.githubusercontent.com/86924581/208297957-f3bd4093-3bb0-49ce-ba9c-cac1a0921ea3.png)

version 10 build 291
dnd5e version 2.0.3

**Comment by Haxxer** [comment]
Is your simple calendar module up to date?

**Comment by Haxxer** [comment]
Besides, the rest button in the player list is for you to prompt rests for players, not for players to rest. They do that through their character sheets.

**Comment by webmaster94** [comment]
I can confirm that this is indeed an error. The module works fine without Simple Calendar running but when Simple Calendar is on, it returns that error when you attempt to prompt for a rest or if a rest is initiated on a character.

**Comment by Haxxer** [comment]
Indeed, I have identified that issue, but for the first error, Simple Calendar needs to be of the latest version.

**Comment by webmaster94** [comment]
Simple Calendar Version: 1.3.75
Hmm..I have no idea why my Simple Calendar has not updated to the latest version, it is not locked but it is not receiving updates. I will need to try and fully re-install it.

**Comment by webmaster94** [comment]
Ok can confirm this appears to be a v10 migration issue. I assume since I am not the only person that has had this issue that it can happen depending on certain configuration settings. It looks like Simple Calendar just gets stuck on the last version for v9. I am surprised it has continued to work just fine up until now.

**Comment by webmaster94** [comment]
Yeah this is explained in Simple Calendar under the release notes for version 2.0.4:

> Update Issue Fix
Right now Foundry says that version 1.3.75 is newer than version 2.0.1 and won't update the module. I will explain why below.

TL;DR: I made a mistake, the quick fix is to uninstall and reinstall Simple Calendar. There will be no data loss and the issue will be solved going forward.

Longer Version: When I first release Simple Calendar I did a silly thing and put a "v" in front of the version that Foundry reads. Foundry uses this version to determine if an update is a newer version or not.

This version is also stored in the module's details so not something that can be fixed without updating the module.

When Foundry compares a version that starts with a v to one that does not start with a v, the one that starts with a v is considered newer. As well the first portion of two versions that both start with a v, example 'v1' and 'v2', are considered to be the same. Until version 2 of Simple Calendar this didn't matter as it was always comparing 'v1' to 'v1'. FoundryVTT won't change how they check version so I need to make the change on my side.

Unfortunately the easiest way to fix this is to uninstall the previous version of Simple Calendar and reinstall it. After doing this, it will update properly going forward. When uninstalling a module that modules' data is not removed so there is no data loss in doing this. That said if you are concerned about data loss please make a backup of your world(s) before doing this exercise.

I apologize about this mistake and the extra work/confusion that may have come because of it.

**Comment by Haxxer** [comment]
The short rest error has been fixed in 1.4.3
